src/main/java/aztech/modern_industrialization/compat/kubejs/machine/RegisterMachinesEventJS.java

```diff
@@ -97,16 +125,98 @@ public void simpleElectricCraftingMultiBlock(
+        extraConfig.accept(config);
+
+        simpleMultiBlock(englishName, internalName, recipeType, multiblockShape, progressBar,
+                itemInputPositions, itemOutputPositions, fluidInputPositions, fluidOutputPositions,
+                controllerCasingName, overlayFolder, frontOverlay, topOverlay, sideOverlay,
+                bep -> new ElectricCraftingMultiblockBlockEntity(bep, internalName, multiblockShape, recipeType),
+                config.reiConfig);
+    }
+
+    public void simpleSteamCraftingMultiBlock(
+            // general
+            String englishName, String internalName, MachineRecipeType recipeType, ShapeTemplate multiblockShape,
+            // REI parameters
+            ProgressBar.Parameters progressBar,
+            Consumer<SlotPositions.Builder> itemInputPositions, Consumer<SlotPositions.Builder> itemOutputPositions,
+            Consumer<SlotPositions.Builder> fluidInputPositions, Consumer<SlotPositions.Builder> fluidOutputPositions,
+            // model
+            String controllerCasingName, String overlayFolder, boolean frontOverlay, boolean topOverlay, boolean sideOverlay) {
+
+        simpleSteamCraftingMultiBlock(englishName, internalName, recipeType, multiblockShape, progressBar,
+                itemInputPositions, itemOutputPositions, fluidInputPositions, fluidOutputPositions,
+                controllerCasingName, overlayFolder, frontOverlay, topOverlay, sideOverlay,
+                config -> {
+                });
+    }
+
+    public void simpleSteamCraftingMultiBlock(
+            // general
+            String englishName, String internalName, MachineRecipeType recipeType, ShapeTemplate multiblockShape,
+            // REI parameters
+            ProgressBar.Parameters progressBar,
+            Consumer<SlotPositions.Builder> itemInputPositions, Consumer<SlotPositions.Builder> itemOutputPositions,
+            Consumer<SlotPositions.Builder> fluidInputPositions, Consumer<SlotPositions.Builder> fluidOutputPositions,
+            // model
+            String controllerCasingName, String overlayFolder, boolean frontOverlay, boolean topOverlay, boolean sideOverlay,
+            // optional config
+            Consumer<ExtraMachineConfig.CraftingMultiBlock> extraConfig) {
+
+        var config = new ExtraMachineConfig.CraftingMultiBlock();
+        config.reiConfig = rei -> rei.steam(true);
+        extraConfig.accept(config);
+
+        simpleMultiBlock(englishName, internalName, recipeType, multiblockShape, progressBar,
+                itemInputPositions, itemOutputPositions, fluidInputPositions, fluidOutputPositions,
+                controllerCasingName, overlayFolder, frontOverlay, topOverlay, sideOverlay,
+                bep -> new SteamCraftingMultiblockBlockEntity(bep, internalName, multiblockShape, recipeType, config.steamOverclockComponent),
+                config.reiConfig);
+    }
+
+    public void simpleMultiBlock(
```

Comment:
Should be `private`

Reply:
Sure. I was thinking maybe it would make some ones life easier who was knowledgeable in kubejs reflection and would allow them to create custom EBF, boilers, etc more easily. They would have to be looking at the source to find it anyway but if the plan is to add things for that in a more friendly way (ie documented) then it probably should be private.